dub.json

```diff
@@ -1,73 +1,118 @@
-			"platforms": ["windows-x86", "posix"],
-			"versions": ["VibeLibeventDriver"],
-			"sourceFiles-windows-x86": ["lib/win-i386/event2.lib"],
-			"copyFiles-windows-x86": ["lib/win-i386/libevent.dll"],
-			"sourceFiles-windows-x86_64": ["lib/win-amd64/event2.lib"],
-			"copyFiles-windows-x86_64": ["lib/win-amd64/libevent.dll"],
-			"libs-windows": ["wsock32", "ws2_32", "advapi32"],
-		},
-		{
-			"name": "libasync",
-			"dependencies": {"libasync": "0.7.1"},
-			"versions": ["VibeUseNativeDriverType", "VibeLibasyncDriver"]
-		},
-		{
-			"name": "libev",
-			"dependencies": {"libev": "~>5.0.0+4.04"},
-			"platforms": ["posix"],
-			"versions": ["VibeLibevDriver", "LIBEV4"],
-		},
-		{
-			"name": "libev-travis",
-			"dependencies": {"libev": "~>5.0.0+4.04"},
-			"platforms": ["posix"],
-			"versions": ["VibeLibevDriver", "LIBEV4"],
-			"lflags": ["-Llibev-4.19/.libs/", "-rpath=libev-4.19/.libs/"],
-		},
-		{
-			"name": "win32",
-			"platforms": ["windows"],
-			"versions": ["VibeWin32Driver"],
-			"libs-windows": ["wsock32", "ws2_32", "advapi32", "user32"],
-		},
-		{
-			"name": "winrt",
-			"platforms": ["windows"],
-			"versions": ["VibeWinrtDriver"],
-			"sourceFiles-windows-x86": ["lib/win-i386/runtimeobject.lib"],
-		}
-	],
-
-	"-ddoxFilterArgs": ["--unittest-examples", "--min-protection=Protected", "--ex", "vibe.core.drivers.", "--ex", "vibe.internal."]
-}
+{
+	"name": "vibe-d",
+	"description": "Asynchronous I/O framework",
+	"license": "MIT",
+	"copyright": "Copyright © 2012-2014 rejectedsoftware e.K.",
+	"homepage": "http://vibed.org/",
+	"authors": [
+		"Sönke Ludwig",
+		"Jan Krüger",
+		"Matthias Dondorff",
+		"see github for more"
+	],
+
+	"dependencies": {
+		"libhttp2": { "version": "~>0.2.3" },
+	},
+
+	"targetType": "library",
+	"mainSourceFile": "source/vibe/appmain.d",
+	"buildRequirements": ["requireBoundsCheck"],
+
+	"configurations": [
+		{
+			"name": "libevent",
+			"systemDependencies": "libevent 2.0.x, OpenSSL 1.0.x",
+			"dependencies": {"libevent": "~>2.0.1+2.0.16","openssl": ">=1.0.0+1.0.0e"},
+			"platforms": ["windows-x86", "posix"],
+			"versions": ["OpenSSL", "VibeLibeventDriver"],
+			"sourceFiles-windows-x86": ["lib/win-i386/event2.lib","lib/win-i386/eay.lib", "lib/win-i386/ssl.lib"],
+			"copyFiles-windows-x86": ["lib/win-i386/libevent.dll","lib/win-i386/libeay32.dll", "lib/win-i386/ssleay32.dll"],
+			"sourceFiles-windows-x86_64": ["lib/win-amd64/event2.lib","lib/win-amd64/libeay32.lib", "lib/win-amd64/ssleay32.lib"],
+			"copyFiles-windows-x86_64": ["lib/win-amd64/libevent.dll","lib/win-amd64/libeay32.dll", "lib/win-amd64/ssleay32.dll"],
+			"libs-windows": ["wsock32", "ws2_32", "advapi32"]
+		},
+		{
+			"name": "libasync_botan",
+			"dependencies": {
+				"botan": {"version": "~>1.11.12" },
+				"libasync": {"version": "~>0.7.3" }
+			},
+			"versions": ["Botan", "VibeUseNativeDriverType", "VibeLibasyncDriver"]
+		},
+		{
+			"name": "libasync_botan_win32",

```

Comment:
Here the -m32mscoff option isn't available as a dub architecture choice and the dflag propagate in the dependencies. I'd need to have some kind of shell script or a new feature in dub. So, this is a temporary workaround (unless someone teaches me something new)